pdf/pdfrenderthread.cpp

```diff
@@ -208,6 +230,13 @@ QMultiMap<int, QPair<QRectF, QUrl> > PDFRenderThread::linkTargets() const
     QMutexLocker(&d->thread->mutex);
     return d->linkTargets;
 }
+const QList< QPair< QRectF, QString > > PDFRenderThread::textBoxesAtPage(int page)
```

Comment:
Empty line here as well. Inside method there are extra spaces on if (), and QMutexLocker has curly brace initialization not used elsewhere.